app/src/main/java/org/simple/clinic/recentpatientsview/RecentPatientsView.kt

```diff
@@ -66,13 +75,19 @@ class RecentPatientsView(
     )
   }
 
+  init {
+    val layoutInflater = LayoutInflater.from(context)
+    binding = RecentPatientsBinding.inflate(layoutInflater, this)
+  }
+
   override fun onFinishInflate() {
     super.onFinishInflate()
+    if (isInEditMode) {
+      return
+    }
 
     context.injector<Injector>().inject(this)
 
-    inflate(context, R.layout.recent_patients, this)
```

Comment:
If we are not inflating the layout file how is the layout getting attached?

Reply:
We are doing that during the `init